pkg/bbgo/interact.go

```diff
@@ -74,6 +56,15 @@ func (it *CoreInteraction) FilterStrategyByInterface(checkInterface interface{})
 	return strategies, found
 }
 
+func (it *CoreInteraction) GetStrategyController(strategy SingleExchangeStrategy) (controller StrategyControllerInterface, found bool) {
```

Comment:
this function does not need the CoreInteraction instance, so you can make it a simple function. (without receiver)

Reply:
something like:

sc, ok := toStrategyController(instance)

instead of:

sc, ok := it.GetStrategyController(instance)

Reply:
this function is no longer needed